src/plugins/fcrypt/testmod_fcrypt.c

```diff
@@ -114,6 +114,28 @@ static int isTestFileCorrect (const char * file)
 	return returnValue;
 }
 
+static int gpg_available (void)
+{
+	int available = 0;
+	char * gpgPath = NULL;
+	KeySet * conf = newPluginConfiguration ();
+	Key * parentKey = keyNew ("system", KEY_END);
+
+	int gpg_search_result = CRYPTO_PLUGIN_FUNCTION (gpgGetBinary) (&gpgPath, conf, parentKey);
+	if (gpg_search_result == 1)
+	{
+		available = 1;
```

Comment:
how does gpg_search_result and available differ?

Reply:
`gpg_search_result` is set to `-1` in case of error.